nodestream/pipeline/extractors/ttls.py

```diff
@@ -7,13 +7,21 @@
 
 class TimeToLiveConfigurationExtractor(Extractor):
     def __init__(
-        self, graph_object_type: str, configurations: Iterable[Dict[str, Any]]
+        self,
+        graph_object_type: str,
+        configurations: Iterable[Dict[str, Any]],
+        global_expiry_in_hours: int = None,
     ) -> None:
         self.graph_object_type = GraphObjectType(graph_object_type)
         self.configurations = configurations
+        self.global_expiry_in_hours = global_expiry_in_hours
 
     async def extract_records(self):
         for config in self.configurations:
+            # Only when global_expiry_in_hours is set, will we have an overwrite of all ttl's
```

Comment:
I think we need to add an explanation similar to this to the documentation on TTLs.